strapi/strapi
# Show installed providers as already installed on marketplace

### What does it do?

Two visible changes on the marketplace:

- Show installed providers as already installed on marketplace
- Show disabled (but installed) plugins as already installed

Note that this PR does not handle packages installed by default by a plugin (local upload and sendmail). They will be removed from the database the day we ship this change.

I also took this opportunity to do a couple technical changes:

- Use the appInformation data from the context, to no longer call that endpoint twice
- Check if a package is installed outside of the card component, for a slight perf benefit (I think)

### Why is it needed?

To let users know they don't need to install a package they already have.

### How to test it?

- disable a plugin via `config/plugins.js` => it should still appear as installed on the marketplace page
- on the marketplace providers tab, some providers should appear as already installed, and they won't have the copy install command button

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request
